tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/InstrumentationVerification.cs

```diff
@@ -54,7 +58,8 @@ private static string GetInstrumentationLogsFolder(Process process, string logsF
 
         if (!instrumentationLogsFolder.Exists)
         {
-            throw new Exception($"Unable to find instrumentation verification directory at {instrumentationLogsFolder}");
+            instrumentationLogsFolder = new DirectoryInfo(@"C:\ProgramData\Datadog-APM\logs\DotNet\InstrumentationVerification");
```

Comment:
Looks like this needs to be reverted or otherwise fixed and verified?

Reply:
Fixed in d6e34e581c70cc2271806c12db2c3d4f2caf78f1